hrms/__init__.py

```diff
@@ -1,6 +1,6 @@
 import frappe
 
-__version__ = "15.33.2"
+__version__ = "15.33.4"
```

Comment:
undo this change. You might need to rebase your commits

Reply:
@ruchamahabal I have reverted the version change